### DISCUSSION TITLE
Make command for notification channel

### DISCUSSION THREAD

**Discussion by MrYamous** [laravel/framework#41679] [Ideas] [answered]
Hello Laravel Community 

As a fairly new Laravel developer, i was using Laravel Notifications for the first time and wanted to use a custom channel. 

While reading related section in the doc, I instinctively ran `php artisan make:channel` with the expectation to be able to use the newly created class to do what i need. But the created class seems to be related to Broadcasting and not to Notifications.

This is probably not a major issue, but what would you think about adding an artisan:make command to create a Channel class for Notifications ?

**Comment by a-bashtannik** [comment, marked as answer]
Notification channels are developed by the community as packages, it's not a daily deal - most of the channels have their custom logic inside, and making one single stub makes no sense. Finally, you will "invent" the interface with `send()` method. Btw, it's weird all notification channels are not using a contract. 

https://laravel-notification-channels.com/
https://github.com/laravel-notification-channels/channels

**Reply by MrYamous** [reply]
Make sens, thank for explanation. I agree about contract
